src/DeviceInterface/overlayApi.js

```diff
@@ -14,8 +17,22 @@ export function overlayApi (device) {
      * page load every time it's opened.
      */
     window.addEventListener('mouseMove', (event) => {
+        // Don't set focus if the mouse hasn't moved ever
+        // This is to avoid clickjacking where an attacker puts the pulldown under the cursor
+        // and tricks the user into clicking
+        if (
+            (!previousX && !previousY) || // if no previous coords
+            (previousX === event.detail.x && previousY === event.detail.y) // or the mouse hasn't moved
+        ) {
+            previousX = event.detail.x
+            previousY = event.detail.y
+            return
+        }
+
```

Comment:
This is the meat of the PR. Basically, it avoids setting the `.currentFocus` class if the mouse has not moved.